src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,27 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type TodoInfoProps = {
+  todo: Todo;
+  user: User | undefined;
+};
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo, user }) => {
+  return (
+    <article
+      key={todo.id}
```

Comment:
The `key` attribute is typically used in list rendering to help React identify which items have changed, are added, or are removed. Since this is not a list rendering scenario, the `key` attribute is unnecessary here and can be removed.